openslides/assignments/static/js/assignments/base.js

```diff
@@ -183,6 +191,25 @@ angular.module('OpenSlidesApp.assignments', [])
                 // subtitle of search result
                 getSearchResultSubtitle: function () {
                     return "Election";
+                },
+                // override project function of jsDataModel factory
+                project: function (poll_id) {
+                    return $http.post(
+                        '/rest/core/projector/1/prune_elements/',
+                        [{name: 'assignments/assignment', id: this.id, poll: poll_id}]
+                    );
+                },
+                // override isProjected function of jsDataModel factory
+                isProjected: function (poll_id) {
+                    // Returns true if there is a projector element with the name
+                    // 'assignments/assignment'.
+                    var projector = Projector.get(1);
+                    if (typeof projector === 'undefined') return false;
```

Comment:
Please do not call a return inside a function. Use an if-else-clause for example.